src/dosinst.c

```diff
@@ -2555,9 +2545,7 @@ command_line_setup_choices(int argc, char **argv)
 	    choices[choice_count - 1].arg = vimfiles_dir_choice;
 	}
 	else if (strcmp(argv[i], "-register-OLE") == 0)
-	{
-	    /* This is always done when gvim is found */
-	}
+	    ; /* This is always done when gvim is found */
```

Comment:
Personally, I prefer the

```c
{
    // comment
}
```

style.
`; /* ... */` doesn't stand out, does it?